tools/c7n_tencentcloud/c7n_tencentcloud/query.py

```diff
@@ -279,7 +287,7 @@ def get_resource_query_params(self):
     def resources(self):
         params = self.get_resource_query_params()
         resources = self.source.resources(params)
-
+        resources = self.augment(resources)
```

Comment:
Is it possible to get `qcs` added to every resource as we  are loading them? Having this in the resource JSON will be very useful. Also, if this becomes part of the standard method, other resources won't need to add their own methods to get the `qcs`.

Reply:
Sure. We will add this attribute to each resource in a next PR.